controllers/update_status.go

```diff
@@ -325,7 +325,7 @@ func tryConnectionOptions(logger logr.Logger, cluster *fdbv1beta2.FoundationDBCl
 		logger.Error(err, "Error getting connection string from cluster", "connectionString", connectionString)
 	}
 
-	return originalConnectionString, err
+	return originalConnectionString, nil
```

Comment:
Not sure why you are reverting this change. It appears we couldn't get connection string from cluster in this case.

Reply:
I removed it to not mix different things together.